src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+
+}
+
+.logo__img {
+  padding: 10px 0;
```

Comment:
Redundant, cause `align-items: center` on parent element will anyway center it vertically